utils/scripts/testconfig/test-variables.yaml

```diff
@@ -7,3 +7,13 @@ OCP_ADMIN_USER:
   AUTH_TYPE: OCP_ADMIN_USER_AUTH_TYPE
   USERNAME: OCP_ADMIN_USER_USERNAME
   PASSWORD: OCP_ADMIN_USER_PASSWORD
+ISV_CRED:
```

Comment:
IMO, we could avoid nesting the credentials unless we plan to iterate through them at some point of time.

Reply:
removed the nested with plain dict